src/python/lib/mesos/http.py

```diff
@@ -20,45 +20,46 @@
 Classes and functions for interacting with the Mesos HTTP restful API
 """
 
-from __future__ import absolute_import
-
+import copy
 import logging
-import urlparse
-
-# requests must be imported after all native packages
 import requests
 import tenacity
 import ujson
+import urlparse
 
-from mesos.exceptions import (HTTPException, MesosAuthenticationException,
-                              MesosAuthorizationException, MesosBadRequest,
-                              MesosUnprocessableException)
+from mesos.exceptions import HTTPException
+from mesos.exceptions import MesosException
 
-METHOD_HEAD = 'HEAD'
-METHOD_GET = 'GET'
-METHOD_POST = 'POST'
-METHOD_PUT = 'PUT'
-METHOD_PATCH = 'PATCH'
-METHOD_DELETE = 'DELETE'
```

Comment:
These act as symbolic constants for the methods, and will be referred to when the method-specific methods like Resource.get() and Resource.post() are added.